src/queries/disputes.js

```diff
@@ -0,0 +1,107 @@
+import gql from 'graphql-tag'
+
+export const AllDisputes = gql`
+  subscription {
+    disputes(orderBy: createdAt, orderDirection: desc) {
+      id
+      createTermId
+      possibleRulings
+      finalRuling
+      lastRoundId
+      state
+      metadata
+      createdAt
+      subject {
+        id
+      }
+      rounds {
+        state
+        number
+        draftTermId
+        jurorsNumber
+        settledPenalties
+        jurorFees
+        delayedTerms
+        selectedJurors
+        coherentJurors
+        collectedTokens
+        createdAt
+        jurors {
+          juror {
+            id
+          }
+        }
+        appeal {
+          id
+          maker
+          appealedRuling
+          taker
+          opposedRuling
+          settled
+          createdAt
+        }
+      }
+    }
+  }
+`
+
+export const JurorDrafts = gql`
+  query JurorDrafts($id: ID!) {
+    juror(id: $id) {
+      id
+      drafts {
+        id
+        weight
+        rewarded
+        commitment
+        outcome
+        leaker
+        createdAt
+        round {
+          id
+          number
+          dispute {
+            id
+            createTermId
+            possibleRulings
+            finalRuling
+            lastRoundId
+            state
+            metadata
+            createdAt
+            subject {
+              id
+            }
+            rounds {
+              state
+              number
+              draftTermId
+              jurorsNumber
+              settledPenalties
+              jurorFees
+              delayedTerms
+              selectedJurors
+              coherentJurors
+              collectedTokens
+              createdAt
+              jurors {
+                juror {
+                  id
+                }
+              }
+              appeal {
+                id
+                maker
+                appealedRuling
+                taker
+                opposedRuling
+                settled
+                createdAt
+              }
+            }
+          }
+        }
+      }
+    }
+  }
```

Comment:
This one sounds reasonable

Reply:
However, maybe it does worth having some lazy components that could trigger more queries if needed. For example, maybe we only need the information on the last round for each dispute on the details page. In case the user clicks on a component that requires information about other rounds, we could go and fetch that data